ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/snmp-settings.j2

```diff
@@ -1,48 +1,48 @@
 {# eos - SNMP Settings#}
-{% if snmp_server is defined and snmp_server is not none %}
+{% if snmp_server is arista.avd.defined %}
 !
-{%     if snmp_server.contact is defined and snmp_server.contact is not none %}
+{%     if snmp_server.contact is arista.avd.defined %}
 snmp-server contact {{ snmp_server.contact }}
 {%     endif %}
-{%     if snmp_server.location is defined and snmp_server.location is not none %}
+{%     if snmp_server.location is arista.avd.defined %}
 snmp-server location {{ snmp_server.location }}
 {%     endif %}
-{%     if snmp_server.ipv4_acls is defined and snmp_server.ipv4_acls is not none %}
+{%     if snmp_server.ipv4_acls is arista.avd.defined %}
 {%         for acl in snmp_server.ipv4_acls %}
-snmp-server ipv4 access-list {{ acl.name }}{% if acl.vrf is defined and acl.vrf is not none %} vrf {{ acl.vrf }} {% endif %}
+snmp-server ipv4 access-list {{ acl.name }}{% if acl.vrf is arista.avd.defined %} vrf {{ acl.vrf }} {% endif %}
```

Comment:
refactor to cli variable

Reply:
ok to do that. I was testing if we can use ``` | arista.avd.natural_sort ``` but seems EOS reorders these commands differently and I don't see any pattern. Any ideas?